build-system/pr-check/utils.js

```diff
@@ -37,14 +37,23 @@ const {execOrDie, execOrThrow, execWithError, exec} = require('../common/exec');
 const {getLoggingPrefix, logWithoutTimestamp} = require('../common/logging');
 const {replaceUrls} = require('../tasks/pr-deploy-bot-utils');
 
-const UNMINIFIED_OUTPUT_FILE = `amp_unminified_${ciBuildSha()}.zip`;
-const NOMODULE_OUTPUT_FILE = `amp_nomodule_${ciBuildSha()}.zip`;
-const MODULE_OUTPUT_FILE = `amp_module_${ciBuildSha()}.zip`;
-const EXPERIMENT_OUTPUT_FILE = (exp) => `amp_${exp}_${ciBuildSha()}.zip`;
+const UNMINIFIED_CONTAINER_DIRECTORY = `unminified`;
```

Comment:
For completeness, may be worth adding an entry for experiments and using it in `uploadExperimentOutput()`?
```js
const EXPERIMENT_CONTAINER_DIRECTORY = (exp) => `experiment${exp}`;
```

Reply:
`exp` is already `experimentA`, not `A`... Seems a bit redundant :))